src/main/java/com/easypost/net/EasyPostResponse.java

```diff
@@ -8,8 +8,6 @@
 	public EasyPostResponse(int responseCode, String responseBody) {
 		this.responseCode = responseCode;
 		this.responseBody = responseBody;
-
-    // System.out.println(this.responseBody);
```

Comment:
This is already being removed from the JSON PR @nwithan8 is doing, you can leave it alone and it'll be take care of separately.